graph_altimeter/scan/__init__.py

```diff
@@ -190,7 +191,7 @@ def _create_vertex(v_id, scan_id):
         "~id": v_id,
```

Comment:
Is this `v_id` fine? You are converting it to `str` below but not here.

Reply:
Yes, it is because Altimeter converts the id's of the vertices to strings before writing them:
https://github.com/manelmontilla/altimeter/blob/master/altimeter/core/neptune/client.py#L705

Reply:
Yes, it is because Altimeter converts the id's of the vertices to strings before writing them:
https://github.com/manelmontilla/altimeter/blob/master/altimeter/core/neptune/client.py#L705